src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md

```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
 
 You can now run the Galasa CLI too from any directory in your file system without having to specify the absolute path.
 
+## Installing Java 
+
+Install a Java version 11 JDK or later. We do not currently support Java 17 or later. The following example uses Homebrew to install Java version 11 on a MacOS.
+
+1. Install Java version 11 JDK using HomeBrew by running the following command in your terminal:
+```
+brew install openjdk@11
+```
+2. Set the JAVA_HOME environment variable to reference the JVM in which you want the test to run. To avoid setting this on every terminal, add the following information to your shell’s initialization file:
+```
+export JAVA_HOME=/path/to/your/jdk/Contents/Home
+```
+where `/path/to/your/jdk` is you JDK path. 
+You can find your JDK path by running the `which java` command in your terminal. 
+
+
+
+## Installing Maven
+
+Install Maven 3.8 or later. The following example uses Homebrew:
+
+1. Run the following command to install Maven:
+```
+brew update
```

Comment:
Brew is a mac-only thing. 
If we tell them how to install it, do we not have to tell windows users, and linux users how to install it ?